docker-compose-services/redis/docker-compose.redis.yaml

```diff
@@ -14,7 +14,7 @@ services:
     volumes:
       - ".:/mnt/ddev_config"
       - "./redis:/usr/local/etc/redis"
-    command: ["redis-server", "/usr/local/etc/redis/redis.conf"]
+    command: ["redis-server", "--bind", "redis", "--port", "6379"]
```

Comment:
This change throws away the redis config that is provided by the user, which is mounted in /usr/local/etc/resdis.